DOC-2987. Added manual appliance update instructions.

rachelbritton opened the pull request:
Added instructions for how to manually download and apply updates to MDDS appliances.

rachelbritton committed:
DOC-2987. Added manual appliance update instructions.

Added instructions for how to manually download and apply updates to MDDS appliances.

eythort-mm commented on guides/implementation/updates.rst:
```diff
@@ -116,6 +116,13 @@ The update process involves downloading and deploying updates reflected in the :
+To update an appliance in Micetro, you can either download and apply the update directly through the Web Application or manually.
+
+.. note::
+   It's recommended to download and update your MDDS appliances through the Web Application.
```
I think it's only recommended because it's easier. But a manually downloaded zip works the same way, so perhaps this statement is not necessary?

eythort-mm commented on guides/implementation/updates.rst:
```diff
@@ -127,3 +134,21 @@ The update process involves downloading and deploying updates reflected in the :
 
 4. Once the download is complete, the update **Status** becomes :guilabel:`Pending`. Initiate the update process by selecting :guilabel:`Deploy` on the Row :guilabel:`...` menu.
+
+**To update an appliance manually**:
```
I'd say this chapter is describing how to manually download an update package to Micetro, but then update the appliance after that the normal way through the web app. It's also possible to update the appliance manually, but that would involve both downloading the package manually and applying it manually by pushing the package to the MDDS and then logging into the appliance and executing the update command (which I believe is documented for BAM).

What I'm trying to say: perhaps this title should be closer to "How to manually download an update package and put it in the right place"

eythort-mm commented on guides/implementation/updates.rst:
```diff
@@ -127,3 +134,21 @@ The update process involves downloading and deploying updates reflected in the :
+
+**To update an appliance manually**:
+
+1. Download the update file (.tgz) from the update server at https://update.menandmice.com/appliance/updates/.
```
Step 1 would actually be to look at the index of appliance updates at https://update.menandmice.com/appliance/index.json and grab the filename for the update from that file e.g. "filename": "bdds-9.6.1.zip"
in Step 2, you would download the file with the filename from step 1,  https://update.menandmice.com/appliance/updates/filename.zip, e.g. https://update.menandmice.com/appliance/updates/bdds-9.6.1.zip. Note that it's a zip file not tgz

eythort-mm commented on guides/implementation/updates.rst:
```diff
@@ -127,3 +134,21 @@ The update process involves downloading and deploying updates reflected in the :
+   .. note::
+      We recommend naming the folder based on the update version, e.g., ``C:\ProgramData\Men and Mice\Central\update\appliance\9.6.1`` or ``/var/mmsuite/mmcentral/update/appliance/9.6.1``.
+
+3. Upload the update package to the Central server.
```
this should be step 2 (or just first part of the unzip step)

eythort-mm commented on guides/implementation/updates.rst:
```diff
@@ -116,6 +116,13 @@ The update process involves downloading and deploying updates reflected in the :
 
+How to Update Appliances
+^^^^^^^^^^^^^^^^^^^^^^^^
+To update an appliance in Micetro, you can either download and apply the update directly through the Web Application or manually.
```
...you can either both download and apply the update through the Web application, or download the update package manually and then apply it through the Web application. 
It's also possible to update MDDS manually by following -link to BAM instructions for that here-

rachelbritton committed:
DOC-2987. Revised based on feedback.

Revised based on feedback and moved the file from implementation folder to admin-manual folder.

rachelbritton committed:
DOC-2987. Adjusted Update Paths section.

Changed the Update Paths section, removing the table and replacing it with basic instructions and important notes for Kea DHCP and Windows.
